ixmp/backend/jdbc.py

```diff
@@ -603,13 +642,10 @@ def init_item(self, s, type, name, idx_sets, idx_names):
         try:
             func(name, idx_sets, idx_names)
         except jpype.JException as e:
-            e = str(e)
-            if 'This Scenario cannot be edited' in e:
-                raise RuntimeError(e)
-            elif 'already exists' in e:
-                raise ValueError('{!r} already exists'.format(name))
+            if 'already exists' in e.args[0]:
+                raise ValueError(f'{name!r} already exists')
```

Comment:
https://www.python.org/dev/peps/pep-0498/#s-r-and-a-are-redundant

Reply:
Ah, nice. We should then have an issue like pandas-dev/pandas#29886 and do all ~50 usages at once.

Reply:
Created #293.